### PR TITLE
PSP-1256 Add new sidebar logic with Add a Property button

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: frontend/src/features/mapSideBar/containers/PimsInventoryContainer.test.tsx [hunk 1]
```diff
@@ -27,7 +26,7 @@ import { networkSlice } from 'store/slices/network/networkSlice';
 import { propertiesSlice } from 'store/slices/properties';
 import { defaultTenant, TenantProvider } from 'tenants';
 
-import MapSideBarContainer from './MapSideBarContainer';
+import PimsInventoryContainer from './PimsInventoryContainer';
```
Comment (Fosol) [inline frontend/src/features/mapSideBar/containers/PimsInventoryContainer.test.tsx:29]: I like the rename

File: frontend/src/utils/TestCommonWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  store?: any;
+  agencies?: IAgency[];
+  roles?: IRole[];
+}
+
+/**
+ * The purpose of this wrapper is to provide mock context provider functionality for common functionality within the project, such as redux, router, etc.
+ * Reduces the amount of boilerplate required for a given test, and allows each test file to focus on test-specific logic.
+ */
+const TestCommonWrapper: React.FunctionComponent<TestProviderWrapperParams> = ({
```
Comment (Fosol) [inline frontend/src/utils/TestCommonWrapper.tsx:23]: like

File: frontend/src/utils/TestCommonWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import { useKeycloak } from '@react-keycloak/web';
+import { IRole } from 'interfaces';
+import { IAgency } from 'interfaces/agency';
+import React from 'react';
+import { TenantProvider } from 'tenants';
+
+import TestProviderWrapper from './TestProviderWrapper';
+import TestRouterWrapper from './TestRouterWrapper';
+
+jest.mock('@react-keycloak/web');
```
Comment (Fosol) [inline frontend/src/utils/TestCommonWrapper.tsx:10]: Wonder if there is a way to also remind the test to clean this up at the end.

**Review by Fosol — APPROVED** [review]

File: frontend/src/utils/TestCommonWrapper.tsx (hunk 3 above)
Comment (devinleighsmith) [reply frontend/src/utils/TestCommonWrapper.tsx:10]: I'm still hoping we turn on the jest flag to reset mock states between tests.

**Review by jdtoombs — APPROVED** [review]

**Pull request merged**